firmware/boron-ins-fsm/src/im21door.cpp

```diff
@@ -70,9 +72,13 @@ doorData checkIM21(){
     // read as: doorLowBatteryFlag is true if doorStatus AND 0b0100 is not 0b0000
     doorLowBatteryFlag = (currentDoorData.doorStatus & (1 << 2)) != 0;
 
+    //record the time a IM21 message was received
+    doorLastMessage = millis();
+
     // Checks if the 3rd bit of doorStatus is 1
     if ((currentDoorData.doorStatus & (1 << 3)) != 0){
       doorHeartbeatReceived = millis();
+      doorHeartbeatFlag = true;
```

Comment:
You're totally right that that was what was written in the Task. Do you think it would end up getting spammy? Hmmm.

What do you think about a maximum number then as well? Like, if it receives any type of door message, if it's been longer than X minutes since the last door message, send a Boron heartbeat right away.

Reply:
ok sounds good